tests/Providers/Stripe/Customer/ServiceTest.php

```diff
@@ -50,9 +51,7 @@ public function testCanAddCard()
             'test'
         );
 
-        $card = $this->service->cards()->find('pm_card_visa');
-
-        $card = $this->service->customers()->addCard($customer, $card);
+        $card = $this->service->customers()->addCard($customer->uid, 'pm_card_visa');
```

Comment:
Why isn't `$this->service` already the CustomerService as you seem to only need that one now?

Reply:
Because if so we'll have to add multiple service via a service provider. So here we decided that the core service which is the PaymentService knows all the other one  and we are able to retrieve it directly from this one.